plugins/_collections.js

```diff
@@ -2289,3 +2289,66 @@ exports.pseudoClasses = {
   userAction: ['hover', 'active', 'focus', 'focus-visible', 'focus-within'],
   functional: ['is', 'not', 'where', 'has'],
 };
+
+/**
+ * @type {string[]}
+ * @see https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute
```

Comment:
```suggestion
 * @see https://developer.mozilla.org/docs/Web/SVG/Attribute
```

Can we remove local specific paths from the link?
It's just so when someone who doesn't want English clicks it, they don't get English.